pages/account/manage/event/[[...data]].js

```diff
@@ -238,6 +249,35 @@ export default function ManageEvent({ BASE_URL, event }) {
                       For example: In <i>DD / MM / YYYY, HH:MM</i>
                     </p>
                   </div>
+                  <div className="mt-1 sm:col-span-2 sm:mt-0">
+                  <div className="mt-1">
+                        <Select
+                          name="userStatus"
+                          label="Are you speaking in the event?"
+                          value={userStatus}
+                          options={DropDownOptions}
+                          onChange={handleEventTypeChange}
+                        />
+                      </div>
+                      <p className="text-sm text-primary-low-medium">
+                        Select Speaking or Non-speaking
+                      </p>
+                    </div>
+                    {userStatus === "Speaking" && (
+                      <div className="mt-1 sm:col-span-2 sm:mt-0">
+                        <Input
+                          type="text"
+                          min="0"
```

Comment:
min 0 is not recommended, if required, used `required`

also a `max` is required, this should match the model max

Reply:
Here is the change:
```
                        <Input
                          type="text"
                          name="topic"
                          label="What topic are you speaking on ?"
                          placeholder= "Your speaking topic"
                          onChange={(e) => setspeakingTopic(e.target.value)}
                          value={speakingTopic}
                          required
                          maxLength="256"
                        />
```

Reply:
Is it required though? What if they are not speaking?

Reply:
If the user selects the speaking option it only appears then else doesn't appear